scapy/contrib/gtp.py

```diff
@@ -192,14 +192,15 @@ def answers(self, other):
         return (isinstance(other, GTPHeader) and
                 self.version == other.version and
                 self.payload.answers(other.payload))
-
-    @classmethod
-    def dispatch_hook(cls, _pkt=None, *args, **kargs):
-        if _pkt and len(_pkt) >= 1:
-            if (struct.unpack("B", _pkt[0])[0] >> 5) & 0x7 == 2:
-                import gtp_v2
-                return gtp_v2.GTPHeader
-        return GTPHeader
+    

```

Comment:
Thanks @gpotter2 , just followed the  http://aruljohn.com/info/ctrlm/ and it worked, but as every line did have a ^M character, it seems like I have altered the whole document... hope that's OK.

All tests passed.

Thanks for your feedback and support.